app.js

```diff
@@ -13,3 +14,135 @@ menuLinks[1].addEventListener('click', mobileMenuT);
+    imageAlt: 'accrobate lady image',
+    canopy: ['Back End Dev', '2022'],
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    courses: ['html', 'css', 'javaScript', 'Ruby on rails'],
+    sourcebtn: '#',
+    linkBtn: '#',
+  },
+  {
+    id: 4,
+    title: 'Multi-Post Stories',
+    subTitle: 'CANOPY',
+    image: './asset/Snapshoot(2).png',
+    imageAlt: 'accrobate lady image',
+    canopy: ['Back End Dev', '2022'],
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    courses: ['html', 'css', 'javaScript', 'Ruby on rails'],
+    sourcebtn: '#',
+    linkBtn: '#',
+  },
+];
+
+for (let i = 0; i < datas.length; i += 1) {
+  myPortfolio.innerHTML += `
+  <div class='all-components' data-popup-start="myPopup-${i}">
+    <div class='first-image'>
+      <img src='${datas[i].image}' alt='${datas[i].imageAlt}' />
+    </div>
+    <div class='tonic-content'>
+      <h1>${datas[i].title}</h1>
+      <div class='tonic-content-canopy'>
+        <p>${datas[i].subTitle}</p>
+        <ul class='tonic-list'>
+          <li>${datas[i].canopy[0]}</li>
+          <li>${datas[i].canopy[1]}</li>
+        </ul>
+      </div>
+      <p>
+        ${datas[i].description}
+      </p>
+      <div class='tonic-courses'>
+        <ul>
+          <li>
+            <a href='#'>${datas[i].courses[0]}</a>
+          </li>
+          <li>
+            <a href='#'>${datas[i].courses[1]}</a>
+          </li>
+          <li>
+            <a href='#'>${datas[i].courses[2]}</a>
+          </li>
+        </ul>
+      </div>
+      <div class='btn'>
+        <button type='submit' data-popup-ref="myPopup-${i}" class="btn-tonic-single">
+          See Project
+        </button>
+      </div>
+    </div>
+  </div>
+  `;
+}
+
+const popupBtn = document.querySelectorAll('.btn-tonic-single');
+
+popupBtn.forEach((btn) => {
+  btn.addEventListener('click', () => {
+    const popupId = btn.getAttribute('data-popup-ref');
+    const popup = document.querySelector(`[data-popup-id='${popupId}']`);
+
+    if (popup !== undefined && popup !== null) {
+      const closeBtns = popup.querySelectorAll('[data-remove-popup]');
+      popup.classList.remove('hidden');
+      popup.scrollIntoView();
+
+      closeBtns.forEach((btn) => {
+        btn.addEventListener('click', () => {
+          popup.classList.add('hidden');
+          const popupStart = document.querySelector(
+            `[data-popup-start='${popupId}',]`,
+          );
+          popupStart.scrollIntoView();
+        });
+      });
+    }
+  });
+});
+
+function inputSave() {
+  const name = document.getElementById('username').value;
+  const email = document.getElementById('email').value;
+  const message = document.getElementById('msg').value;
+
+  localStorage.setItem('username', name);
+  localStorage.setItem('email', email);
+  localStorage.setItem('msg', message);
+}
+inputSave();
```

Comment:
- [ ] Kindly read and understand perfectly the project requirements. I know you can do it :muscle: :muscle:. Watch this [video](https://www.youtube.com/watch?v=C3tiI3akFq8), or [this](https://www.youtube.com/watch?v=9n3qAeaBMGU). They can help you.
![requirement](https://user-images.githubusercontent.com/65068771/152641191-c35b8432-c270-4002-8353-813070613337.png)